onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -747,16 +749,31 @@ def aten_clamp(
     raise NotImplementedError()
 
 
-def aten_clamp_max(self: TensorType, max: float) -> TensorType:
+def aten_clamp_max_scalar(self, max_):
```

Comment:
Is it possible to validate the shape of max_ to know it is a scalar or tensor instead of having such information in the function name?

Reply:
I haven't thought of a good way. Any suggestions?

Reply:
Ah, for validation, we can specify it in the signature / via some data structure. We can discuss today.

Reply:
They still need to be two functions though.